### PR TITLE
improved FAQ description text

### LINKED ISSUE #706: [📝 Docs]: Review the content of the FAQ page of the release 3
### What Docs changes are you proposing?

The purpose of this issue if the make a content review of the new FAQ page available in the release 3 branch and proceed with the improvements if required.

https://web-release-3.website-2v2.pages.dev/overview/faq

Code is available in /pages/overview/faq

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by kwennB**
**What kind of change does this PR introduce?**

Documentation Enhancement

**Issue Number:**

-  Closes #706 

-  Others? 
I also reviewed the general question texts, which are properly formatted and good to go.

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

**Review by DhairyaMajmudar — COMMENTED** [review]
Here pls. add one more sentence in this paragraph since there's a bug related to faq page width which is fixed by width of two-liner paragraph.

**Comment by kwennB** [comment]
> Here pls. add one more sentence in this paragraph since there's a bug related to faq page width which is fixed by width of two-liner paragraph.

Alright, I'll update this.

**Review by benjagm — CHANGES_REQUESTED** [review]
The description looks good, however to close this issue we need to review the wording and completeness of each question and answer. 

The data fetched for the questions is available in the file:
/data/faq.json

**Comment by kwennB** [comment]
> The description looks good, however to close this issue we need to review the wording and completeness of each question and answer.
> 
> The data fetched for the questions is available in the file: /data/faq.json

Noted, thank you.

File: data/faq.json [hunk 1]
```diff
@@ -2,49 +2,49 @@
   {
     "id": 1,
     "question": "What is JSON Schema?",
-    "answer": "JSON Schema is a vocabulary that allows you to annotate and validate JSON documents. It defines the structure, data types, and constraints of JSON data.",
+    "answer": "JSON Schema is a specification for defining the structure of JSON data. It provides a standardized way to annotate, describe, and validate JSON data. JSON Schema defines content, structure, data types, and other expected constraints within a JSON document. The key aspects of JSON Schema include defining data structure, validation documentation data contracts, and diverse use cases.",
```
Comment (benjagm) [inline data/faq.json:5]:
```suggestion
    "answer": "JSON Schema is a specification for defining the structure of JSON data. It provides a standardized way to annotate, describe, and validate JSON data. JSON Schema defines content, structure, data types, and other expected constraints within a JSON document.",
```

File: data/faq.json [hunk 2]
```diff
@@ -2,49 +2,49 @@
...
     "category": "general"
   },
   {
     "id": 2,
     "question": "How do I create a simple JSON Schema?",
-    "answer": "You can create a simple JSON Schema using a JSON object with properties like 'type', 'properties', and 'required'. These define the data type, properties, and mandatory fields of your JSON document.",
+    "answer": "You create a simple JSON Schema by defining keywords such as 'schema', 'id', 'title', 'description', and 'type', which specify the version, unique identifier, title, and, description of your schema. Then, specify the data type for your schema will validate (e.g., object, array) and define the necessary properties, including their types and constraints. You can also include keywords like; 'properties', 'required', and 'default'. These define the constraints of your JSON document.",
```
Comment (benjagm) [inline data/faq.json:11]:
```suggestion
    "answer": "You can create a simple JSON Schema using a JSON object with properties like 'type', 'properties', and 'required'. These define the data type, properties, and mandatory fields of your JSON document.",
```

Let's keep this one simple.

File: data/faq.json [hunk 3]
```diff
@@ -2,49 +2,49 @@
...
     "category": "general"
   },
   {
     "id": 4,
     "question": "How can I define default values in a JSON Schema?",
-    "answer": "You can use the 'default' keyword to set a default value for a property in your JSON Schema. It provides a fallback value if the property is not present in the JSON document.",
+    "answer": "You can define default values for properties using the 'default' keyword, which provides a fallback value if the property is not present in the JSON document. This feature is useful for simplifying client-side code, generating documentation, creating forms, and mock data for testing, for example, in the schema {'type': 'object', 'properties': {'name': {'type': 'string', 'default': 'John Doe'}, 'age': {'type': 'integer', 'default': 30}}}, the 'name' and 'age' properties will default to 'John Doe' and 30 if not specified.",
```
Comment (benjagm) [inline data/faq.json:23]:
```suggestion
    "answer": "You can use the 'default' keyword to set a default value for a property in your JSON Schema. It provides a fallback value if the property is not present in the JSON document.",
```

**Review by benjagm — CHANGES_REQUESTED** [review]
Great work Blessing!! I just left some suggestions.

File: pages/overview/faq/index.page.tsx [hunk 4]
```diff
@@ -15,8 +15,9 @@ export default function Content() {
       </Head>
       <Headline1>{newTitle}</Headline1>
       <p>
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Nobis, totam?
-        Asperiores delectus perspiciatis, reprehenderit, harum quasi unde
+        If you are new to JSON Schema this section will help you learn more
+        about JSON Schema and JSON data. Below you'll find answers to questions
+        we get asked the most about JSON Schema.
```
Comment (benjagm) [inline pages/overview/faq/index.page.tsx:20]:
```suggestion
        Below you'll find answers to questions we get asked the most about JSON Schema.
```

**Review by benjagm — APPROVED** [review]
Great work!

**Pull request merged**